mtp_noms_ops/templates/security/checks_list.html

```diff
@@ -13,11 +13,11 @@ <h1 class="heading-xlarge">{{ view.title }}</h1>
 
   {% include 'mtp_common/includes/message_box.html' %}
 
-  <p>
+  <p class="font-large">
```

Comment:
i think this should be `lede` not `font-large`, but i'm not concerned if it looks the same. anyway, this part will likely be iterated a bit in the next few weeks